clang/test/Analysis/Checkers/WebKit/unchecked-members.cpp

```diff
@@ -0,0 +1,53 @@
+// RUN: %clang_analyze_cc1 -analyzer-checker=alpha.webkit.NoUncheckedPtrMemberChecker -verify %s
+
+#include "mock-types.h"
+#include "mock-system-header.h"
```

Comment:
Do you need to include this one everywhere? Isn't it just a tiny test to confirm that some specific checker doesn't do something stupid in a system header?

Reply:
Yeah, I guess we don't need to include "mock-system-header.h" here.